jekyll/faq.md

```diff
@@ -10,22 +10,15 @@ css_class: faq
 
 ## Why yet another programming language?
 
-Nim is one of the very few _programmable_ statically typed languages, and one
-of the even fewer that produces native binaries that require no runtime or
-interpreter.
+Nim is one of the very few _programmable_ statically typed languages, and
+combines the speed and memory efficency of C, an expressive syntax,
+memory safety and multiple target languages.
 
+## How fast is Nim?
 
-## What have been the major influences in the language's design?
-
-The language borrows heavily from (in order of impact): Modula 3, Delphi,
-Ada, C++, Python, Lisp, Oberon.
-
-
-## How is Nim licensed?
-
-The Nim compiler and the library are MIT licensed.
-This means that you can use any license for your own programs developed with
-Nim.
+The only overhead Nim has over C is the GC which has been tuned for years and
```

Comment:
I would still simply remove this sentence, you said yourself it's bad for marketing. It's also wrong, "overhead over C" might not be the case, Nim uses thread local heaps, C a shared memory heap by default.